app-admin/schaufel/schaufel-0.8.ebuild

```
@@ -0,0 +1,65 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit user
+
+DESCRIPTION="shovel data around"
+HOMEPAGE="https://github.com/adjust/schaufel"
+EGIT_REPO_URI="https://github.com/adjust/schaufel.git"
+
+LICENSE="MIT"
+SLOT="0"
+
+IUSE="doc"
+
+if [[ ${PV} == 9999 ]]
+then
+	inherit git-r3
+	KEYWORDS=""
+else
+	inherit vcs-snapshot
+	SRC_URI="https://github.com/adjust/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
```

```suggestion
	KEYWORDS="amd64"
```
I think you can mark the package as stable for amd64 straight away.